packages/edge-login-ui-rn/src/common/locales/strings/enUS.json

```diff
@@ -134,7 +134,8 @@
   "create_account_error_message": "An error occurred creating your account. This may have been due to a slow network connection or network interruption. Please retry and if you receive another error that the account already exists, please restart the app and login with your username and password.",
   "try_again": "Try Again",
   "otp_reset_modal_header": "Secure Your Account",
-  "otp_reset_modal_message": "It is highly recommended that you enable Two Factor Authentication (2FA) to protect against unauthorized access to your account. Would you like to enable it now?",
+  "otp_reset_modal_message": "Two factor authentication (2FA) prevents unauthorized access from other devices, even if your username and password is compromised. You can scan a QR code or type in an authentication code to seamlessly authorize other devices. Would you like to enable 2fa?",
```

Comment:
Capitalize "2FA"

Reply:
updated

Reply:
Second "2fa" instance is still lower case.